Only setting one of `index` or `rpmChannel` is not enough

**Describe the bug**

Only setting either one of `index` or `rpmChannel` gives this error: `must have one of index or rpmChannel, must be >= 1`.

**To Reproduce**

1. `$ fan2go detect` (I'm only posting relevant sections here):
   <details>
   <summary>Click me to expand</summary>
   
   ```shell
   > dell_smm-isa-0000
     Fans     Index  Channel  Label        RPM  PWM  Auto 
     -----------------------------------------------------
              1      1        hwmon4/fan1  0    0    false
              2      2        hwmon4/fan2  0    0    false
     Sensors  Index  Label                       Value
     -------------------------------------------------
              1      hwmon4/temp1 (temp1_input)  44000
              2      hwmon4/temp2 (temp2_input)  37000
              3      hwmon4/temp3 (temp3_input)  38000
              4      hwmon4/temp4 (temp4_input)  32000
              5      hwmon4/temp5 (temp5_input)  38000
              6      hwmon4/temp6 (temp6_input)  38000
              7      hwmon4/temp7 (temp7_input)  37000
   
   > coretemp-isa-0000
     Sensors  Index  Label                       Value
     -------------------------------------------------
              1      Package id 0 (temp1_input)  53000
              2      Core 0 (temp2_input)        51000
              3      Core 1 (temp3_input)        51000
              4      Core 2 (temp4_input)        52000
              5      Core 3 (temp5_input)        50000
              6      Core 4 (temp6_input)        52000
              7      Core 5 (temp7_input)        48000
   ```
   </details>

2. Case 1: setting only `rpmChannel` (and also `pwmChannel`)
   a. `$ cat /etc/fan2go/fan2go.yaml`:
      <details>
      <summary>Click me to expand</summary>
   
      ```yaml
      fans:
        - id: fan-1
          hwmon:
            platform: dell_smm-isa-0000
            rpmChannel: 1
            pwmChannel: 1
          neverStop: true
          curve: fan-curve-1
      sensors:
        - id: cpu-package
          hwmon:
            platform: coretemp-isa-0000
            index: 1
      curves:
        - id: fan-curve-1
          linear:
            sensor: cpu-package
            min: 50
            max: 90
      ```
      </details>

   b. `# fan2go fan init -i fan-1`, I got
      <details>
      <summary>Click me to expand</summary>
   
      ```shell
      INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
      INFO  Using persistence at: /etc/fan2go/fan2go.db
      INFO  Deleting existing data for fan 'fan-1'...
      INFO  Computing pwm map...
      ERROR   Unable to set Fan Mode of 'fan-1' to "1": PWM mode stuck to -1
      ERROR   Unable to set Fan Mode of 'fan-1' to "0": write
          /sys/devices/platform/dell_smm_hwmon/hwmon/hwmon5/pwm1_enable:
          invalid argument
      INFO  Measuring RPM curve...
      ERROR   Unable to set Fan Mode of 'fan-1' to "1": PWM mode stuck to -1
      ERROR   Unable to set Fan Mode of 'fan-1' to "0": write
          /sys/devices/platform/dell_smm_hwmon/hwmon/hwmon5/pwm1_enable:
          invalid argument
      WARNING  Could not enable manual fan mode on fan-1, trying to
          continue anyway...
      SUCCESS  Done!
      INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
      WARNING  Cannot send notification, unable to detect user of current display session
      FATAL   fan fan-1: must have one of index or rpmChannel, must be >= 1
      panic:
      
      goroutine 1 [running]:
      github.com/pterm/pterm.checkFatal(...)
            github.com/pterm/pterm@v0.12.61/prefix_printer.go:351
      github.com/pterm/pterm.(*PrefixPrinter).Printfln(0x55e9b8d99dc0, {0xc00050fb40, 0x3d}, {0x0, 0x0, 0x0})
            github.com/pterm/pterm@v0.12.61/prefix_printer.go:293 +0x1ed
      github.com/markusressel/fan2go/internal/ui.Fatal({0xc00050fb40, 0x3d}, {0x0, 0x0, 0x0})
            github.com/markusressel/fan2go/internal/ui/logging.go:52 +0x8b
      github.com/markusressel/fan2go/cmd/fan.glob..func1(0x55e9b8d99e80?, {0x55e9b872711f?, 0x5?, 0x0?})
            github.com/markusressel/fan2go/cmd/fan/curve.go:27 +0xe5
      github.com/markusressel/fan2go/cmd/fan.glob..func2(0x55e9b8d9e860?, {0xc00029b420?, 0x2?, 0x2?})
            github.com/markusressel/fan2go/cmd/fan/init.go:55 +0x2f2
      github.com/spf13/cobra.(*Command).execute(0x55e9b8d9e860, {0xc00029b400, 0x2, 0x2})
            github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
      github.com/spf13/cobra.(*Command).ExecuteC(0x55e9b8d9d160)
            github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
      github.com/spf13/cobra.(*Command).Execute(...)
            github.com/spf13/cobra@v1.7.0/command.go:992
      github.com/markusressel/fan2go/cmd.Execute()
            github.com/markusressel/fan2go/cmd/root.go:88 +0xf9
      main.main()
            ./main.go:25 +0x17
      ```
      </details>

2. Case 2: setting only `index`
   a. `$ cat /etc/fan2go/fan2go.yaml`:
      <details>
      <summary>Click me to expand</summary>
   
      ```yaml
      fans:
        - id: fan-1
          hwmon:
            platform: dell_smm-isa-0000
            index: 1
          neverStop: true
          curve: fan-curve-1
      sensors:
        - id: cpu-package
          hwmon:
            platform: coretemp-isa-0000
            index: 1
      curves:
        - id: fan-curve-1
          linear:
            sensor: cpu-package
            min: 50
            max: 90
      ```
      </details>

   b. `# fan2go fan init -i fan-1`, I got
      <details>
      <summary>Click me to expand</summary>
   
      ```shell
      INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
      INFO  Using persistence at: /etc/fan2go/fan2go.db
      INFO  Deleting existing data for fan 'fan-1'...
      INFO  Computing pwm map...
      ERROR   Unable to set Fan Mode of 'fan-1' to "1": PWM mode stuck to -1
      ERROR   Unable to set Fan Mode of 'fan-1' to "0": write
          /sys/devices/platform/dell_smm_hwmon/hwmon/hwmon5/pwm1_enable:
          invalid argument
      INFO  Measuring RPM curve...
      ERROR   Unable to set Fan Mode of 'fan-1' to "1": PWM mode stuck to -1
      ERROR   Unable to set Fan Mode of 'fan-1' to "0": write
          /sys/devices/platform/dell_smm_hwmon/hwmon/hwmon5/pwm1_enable:
          invalid argument
      WARNING  Could not enable manual fan mode on fan-1, trying to
          continue anyway...
      SUCCESS  Done!
      INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
      WARNING  Cannot send notification, unable to detect user of current display session
      FATAL   fan fan-1: must have one of index or rpmChannel, must be >= 1
      panic:
      
      goroutine 1 [running]:
      github.com/pterm/pterm.checkFatal(...)
            github.com/pterm/pterm@v0.12.61/prefix_printer.go:351
      github.com/pterm/pterm.(*PrefixPrinter).Printfln(0x5636519a8dc0, {0xc0004a9800, 0x3d}, {0x0, 0x0, 0x0})
            github.com/pterm/pterm@v0.12.61/prefix_printer.go:293 +0x1ed
      github.com/markusressel/fan2go/internal/ui.Fatal({0xc0004a9800, 0x3d}, {0x0, 0x0, 0x0})
            github.com/markusressel/fan2go/internal/ui/logging.go:52 +0x8b
      github.com/markusressel/fan2go/cmd/fan.glob..func1(0x5636519a8e80?, {0x56365133611f?, 0x5?, 0x0?})
            github.com/markusressel/fan2go/cmd/fan/curve.go:27 +0xe5
      github.com/markusressel/fan2go/cmd/fan.glob..func2(0x5636519ad860?, {0xc000136260?, 0x2?, 0x2?})
            github.com/markusressel/fan2go/cmd/fan/init.go:55 +0x2f2
      github.com/spf13/cobra.(*Command).execute(0x5636519ad860, {0xc000136240, 0x2, 0x2})
            github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
      github.com/spf13/cobra.(*Command).ExecuteC(0x5636519ac160)
            github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
      github.com/spf13/cobra.(*Command).Execute(...)
            github.com/spf13/cobra@v1.7.0/command.go:992
      github.com/markusressel/fan2go/cmd.Execute()
            github.com/markusressel/fan2go/cmd/root.go:88 +0xf9
      main.main()
            ./main.go:25 +0x17
      ```
      </details>

**Expected behavior**

No error messages.

**Desktop (please complete the following information):**

- Distro: Arch Linux
- `uname -a`: 6.3.2-arch1-1
- `sensors -v`: sensors version 3.6.0+git with libsensors version 3.6.0+git
- `fan2go version`: commit c20619ea88909b5adcaf0ee8084929ef60f6b8d3

I also face similar issue when running `sudo fan2go fan init -i fan_cpu`:

<details> 
  <summary>Log</summary>

```
mks@vostro:~$ sudo fan2go fan init -i fan_cpu
 INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
 INFO  Using persistence at: /etc/fan2go/fan2go.db
 INFO  Deleting existing data for fan 'fan_cpu'...
 INFO  Computing pwm map...
 INFO  Measuring RPM curve...
 SUCCESS  Done!
 INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
 WARNING  Cannot send notification, missing env variable 'DISPLAY'!
  FATAL   fan fan_cpu: must have one of index or rpmChannel, must be >= 1
panic: 

goroutine 1 [running]:
github.com/pterm/pterm.checkFatal(...)
	/home/mks/go/pkg/mod/github.com/pterm/pterm@v0.12.66/prefix_printer.go:351
github.com/pterm/pterm.(*PrefixPrinter).Printfln(0x10e8020, {0xc0004d3440, 0x3f}, {0x0, 0x0, 0x0})
	/home/mks/go/pkg/mod/github.com/pterm/pterm@v0.12.66/prefix_printer.go:293 +0x1ed
github.com/markusressel/fan2go/internal/ui.Fatal({0xc0004d3440, 0x3f}, {0x0, 0x0, 0x0})
	/home/mks/Projects/fan2go/internal/ui/logging.go:52 +0x8b
github.com/markusressel/fan2go/cmd/fan.glob..func1(0x10e80e0?, {0xbba265?, 0x5?, 0x0?})
	/home/mks/Projects/fan2go/cmd/fan/curve.go:27 +0xe5
github.com/markusressel/fan2go/cmd/fan.glob..func2(0x10ecb40?, {0xc000078280?, 0x2?, 0x2?})
	/home/mks/Projects/fan2go/cmd/fan/init.go:55 +0x2f2
github.com/spf13/cobra.(*Command).execute(0x10ecb40, {0xc000078260, 0x2, 0x2})
	/home/mks/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x10eb440)
	/home/mks/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/home/mks/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/markusressel/fan2go/cmd.Execute()
	/home/mks/Projects/fan2go/cmd/root.go:88 +0xf9
main.main()
	/home/mks/Projects/fan2go/main.go:25 +0x17
```

</details>

<details>
  <summary>My config</summary>

```
fans:
  - id: fan_cpu
    hwmon:
      platform: dell_smm-isa-0000
      rpmChannel: 1
      pwnChannel: 1
    neverStop: true
    curve: curve_average
sensors:
  - id: sensor_cpu1
    hwmon:
      platform: dell_smm-isa-0000
      index: 1
  - id: sensor_cpu2
    hwmon:
      platform: dell_smm-isa-0000
      index: 2
  - id: sensor_other
    hwmon:
      platform: dell_smm-isa-0000
      index: 3
curves:
  - id: curve_cpu1
    linear:
      sensor: sensor_cpu1
      min: 30
      max: 60
  - id: curve_cpu2
    linear:
      sensor: sensor_cpu2
      min: 30
      max: 60
  - id: curve_other
    linear:
      sensor: sensor_other
      min: 30
      max: 60
  - id: curve_average
    function:
      type: average
      curves:
        - curve_cpu1
        - curve_cpu2
        - curve_other

```

</details>

And I also don't have any issues (and any curves) when running the `sudo fan2go fan curve -i` command.

```
mks@vostro:~$ sudo fan2go fan curve -i curve_average
 INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
mks@vostro:~$ sudo fan2go fan curve -i curve_cpu1
 INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
mks@vostro:~$ sudo fan2go fan curve -i curve_cpu2
 INFO  Using configuration file at: /etc/fan2go/fan2go.yaml
mks@vostro:~$ sudo fan2go fan curve -i curve_other
 INFO  Using configuration file at: /etc/fan2go/fan2go.yaml

```

--- a/internal/configuration/config.go
+++ b/internal/configuration/config.go
@@ -119,6 +119,7 @@ func GetFilePath() string {
 
 func LoadConfig() {
 	// load default configuration values
+	CurrentConfig = Configuration{}
 	err := viper.Unmarshal(&CurrentConfig)
 	if err != nil {
 		ui.Fatal("unable to decode into struct, %v", err)

